packages/cron-job/CronJobExecutionFactory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Draw\Component\CronJob;
+
+use Draw\Component\CronJob\Entity\CronJob;
+use Draw\Component\CronJob\Entity\CronJobExecution;
+
+class CronJobExecutionFactory
+{
+    public function create(CronJob $cronJob, bool $force = false): CronJobExecution
```

Comment:
No need to do a factory for that just add a method on the CronJob object

CronJob->newExecution(bool $force): CronJobExecution